deploy/017-update-pool-registry.ts

```diff
@@ -0,0 +1,29 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { getConfig } from "../helpers/deploymentConfig";
+import { toAddress } from "../helpers/deploymentUtils";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy, catchUnknownSigner } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const { preconfiguredAddresses } = await getConfig(hre.network.name);
+  const proxyOwnerAddress = await toAddress(preconfiguredAddresses.NormalTimelock || "account:deployer", hre);
+
+  await catchUnknownSigner(
+    deploy("PoolRegistry", {
```

Comment:
This likely needs a new name. When running the deployment scripts this is reusing a previous implementation

Reply:
I think the name is correct because it should reuse the proxy contract for the PoolRegistry and redeploy a new implementation (because the code has changed). I tested it locally on testnet:

<img width="1335" alt="Screenshot 2024-01-19 at 12 10 20" src="https://github.com/VenusProtocol/isolated-pools/assets/366222/71d84d13-fbd4-4e82-b1a0-d59538df2636">

Reply:
Ok, I see what was going on, I was testing locally where the code would be the same.